snxcore/src/platform/linux/xfrm.rs

```diff
@@ -352,8 +352,10 @@ impl XfrmConfigurator {
 
         subnets.retain(|s| !s.contains(&self.dest_ip));
 
+        let ignore_routes = self.tunnel_params.ignore_routes.clone();
```

Comment:
Cloning is not necessary, you can just pass it directly as `&self.tunnel_params.ignore_routes`.